cmake/FindOSGPlugins.cmake

```diff
@@ -28,10 +28,14 @@ foreach(_library ${OSGPlugins_FIND_COMPONENTS})
     set(_component OSGPlugins_${_library})
 
     set(${_library_uc}_DIR ${OSGPlugins_LIB_DIR}) # to help function osg_find_library
-    set(_saved_lib_prefix ${CMAKE_FIND_LIBRARY_PREFIXES}) # save CMAKE_FIND_LIBRARY_PREFIXES
-    set(CMAKE_FIND_LIBRARY_PREFIXES "") # search libraries with no prefix
+    if (NOT MINGW)
+        set(_saved_lib_prefix ${CMAKE_FIND_LIBRARY_PREFIXES}) # save CMAKE_FIND_LIBRARY_PREFIXES
+        set(CMAKE_FIND_LIBRARY_PREFIXES "") # search also libraries with no prefix
+    endif()
```

Comment:
This looks sketchy to me. Either the currently set prefixes should be used, or they shouldn't. Why do you not want to clear them for MinGW specifically, but do clear them for everything else?

Reply:
I am actually wondering why this is here in the first place, because theoretically this would break on default on Linux, as lib is always used as prefix.

MinGW follows Unix conventions, and with the current toolchain all osgPlugins get prefixed with `lib`, so it breaks with `CMAKE_FIND_LIBRARY_PREFIXES` set to ""

Reply:
Looks like FindOSGPlugins.cmake is only used for static builds and OSX deployment. I can't really see why it should be cleared either since the libs should have the default prefix.

Reply:
Yeah, I could put it under a flag, if you're not fine with this.

Reply:
I think I tried meddling with this at one point, but can't remember if I worked out why it needed to be how it is in order to work. I might be able to remember stuff if prompted.